.gitignore

```diff
@@ -15,7 +15,9 @@
 # User-specific stuff:
 .idea/workspace.xml
 .idea/tasks.xml
+.idea/findbugs-idea.xml
```

Comment:
Maybe we then should just commit the changed line with the groovy regarding style changes even though we won't need them. Just to not have this on the commit overview every time.